modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjsp

```diff
@@ -25,11 +28,23 @@ PortalUtil.getCurrentAndAncestorSiteGroupIds(themeDisplay.getScopeGroupId());
 DLURLHelperUtil.getDownloadURL(null, null, null, null, null, null);
 DLURLHelperUtil.getImagePreviewURL(null, null);
 
+emailAddress.getListTypeId();
+emailAddress.setListTypeId();
+
 HtmlParserUtil.extractText(null);
 
 JSONUtil.isJSONObject("test123");
 
+orgLabor.getListTypeId();
+orgLabor.setListTypeId();
+
+phone.getListTypeId();
+phone.setListTypeId();
+
 ConfigurationProviderUtil.getPortletInstanceConfiguration(clazz, themeDisplay);
 
+website.getListTypeId();
+website.setListTypeId();
+
```

Comment:
We will have the same problem as before, classNames doesn't work in JSP files, you could create a second ticket as a backlog for when this type of validation is possible. And link with this one. Keep the previous description and add the context that it should also cover JSP files.

Reply:
@rubenssoares-br

Reply:
Ticket created here: https://liferay.atlassian.net/browse/LPD-37607

Reply:
@nicolas-sss